waku/protocol/v2/waku_store.nim

```diff
@@ -0,0 +1,130 @@
+import chronos, chronicles
+import ./filter
+import tables
+import libp2p/protocols/pubsub/pubsub,
+       libp2p/protocols/pubsub/pubsubpeer,
+       libp2p/protocols/pubsub/floodsub,
+       libp2p/protocols/pubsub/gossipsub,
+       libp2p/protocols/pubsub/rpc/[messages, protobuf],
+       libp2p/protocols/protocol,
+       libp2p/protobuf/minprotobuf,
+       libp2p/stream/connection
+
+import metrics
+
+import stew/results
+
+const
+  WakuStoreCodec* = "/vac/waku/store/2.0.0-alpha2"
+
+type
+  StoreRPC* = object
+    query*: seq[HistoryQuery]
```

Comment:
I'm not quite sure I understand this data structure. There's a sequence of history queries and sequence of response? How do they relate with each other?

A request ID would be nice too, so it is clear that a response is to a specific query.

Reply:
Ah I see now, this is literally this protobuf RPC: https://specs.vac.dev/specs/waku/waku-v2.html#protobuf-1

Reply:
Yes it is.